dpcpp/CMakeLists.txt

```diff
@@ -66,8 +66,11 @@ target_sources(ginkgo_dpcpp
 ginkgo_compile_features(ginkgo_dpcpp)
 target_compile_definitions(ginkgo_dpcpp PRIVATE GKO_COMPILING_DPCPP)
 
+# -sycl-std=2017 is temporary workaround after 2021.4 to propagate subgroup setting correctly
+set(GINKGO_DPCPP_FLAGS "-sycl-std=2017")
```

Comment:
Wouldn't it be better to put this into CMakeLists.txt:70, like what we do for the fp-model.

Reply:
After discussion, the sycl-std also affects some USM/buffer usage, so it is global.
Move to `GINKGO_COMPILER_FLAGS`